test_runner/scenario_test_runner/launch/scenario_test_runner.launch.py

```diff
@@ -33,7 +33,7 @@
 
 
 def architecture_types():
-    return ["awf/universe", "awf/universe/20230906"]
+    return ["awf/universe", "awf/universe/20230906", "awf/universe/20240605"]
```

Comment:
Please write here as a comment on what you wrote in the “architecture_type design” section of this pull request description about the differences between each architecture_type.

Reply:
```suggestion
    # awf/universe:          autoware_auto_perception_msgs/TrafficSignalArray for traffic lights
    # awf/universe/20230906: autoware_perception_msgs/TrafficSignalArray for traffic lights
    # awf/universe/20240605: autoware_perception_msgs/TrafficLightGroupArray for traffic lights
    return ["awf/universe", "awf/universe/20230906", "awf/universe/20240605"]
```